src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -324,6 +321,12 @@
   <data name="SerializerOptionsImmutable" xml:space="preserve">
     <value>Serializer options cannot be changed once serialization or deserialization has occurred.</value>
   </data>
+  <data name="StreamNotWritable" xml:space="preserve">
+    <value>Stream was not writable.</value>
```

Comment:
Nit: was => is ?

Reply:
Looks like I borrowed it from the wrong place:
https://github.com/dotnet/corefx/blob/c390ce7df50252e11f5d322276e9d19e046d1332/src/System.Security.Cryptography.Primitives/src/Resources/Strings.resx#L73

We aren't consistent here: https://github.com/dotnet/corefx/search?utf8=%E2%9C%93&q=stream+was+not+writable&type=

However, I will follow what StreamWriter does and change to is :)

https://docs.microsoft.com/en-us/dotnet/api/system.io.streamwriter.-ctor?view=netframework-4.8#System_IO_StreamWriter__ctor_System_IO_Stream_